endpoints/events/event.go

```diff
@@ -0,0 +1,363 @@
+package events
+
+import (
+	"context"
+	"fmt"
+	"github.com/julienschmidt/httprouter"
+	"github.com/prebid/prebid-server/analytics"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/stored_requests"
+	"net/http"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+// EventType enumerates the values of events Prebid Server can receive for an ad.
+type EventType string
+
+// Possible values of  vents Prebid Server can receive for an ad.
+const (
+	Win EventType = "win"
+	Imp EventType = "imp"
+)
+
+// EventFormat enumerates the values of a Prebid Server event.
+type Format string
```

Comment:
Should this be `ResponseFormat` ? it specifies what kind of response is expected.

Reply:
I agree with your reasoning but I followed PBS Java's naming convention:
https://github.com/rubicon-project/prebid-server-java/blob/master/src/main/java/org/prebid/server/events/EventRequest.java#L32

I'll leave that up to you guys.

Reply:
I'm good with `ResponseFormat` as it's a bit more descriptive. This name doesn't affect the API. We should certainly look at PBS-Java's implementation for guidance on this feature, but I don't see a need to keep the internal names verbatim.